student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom, age, name: 'Unknown', parent_permission: true)
+    super(age, name: name, parent_permission: parent_permission)
```

Comment:
- [ ] you passed in name & parent_permission parameters to your super function like this using variable names, doing this means you are setting a default value with the variable names but there is no where in your code that those variable names have been defined and as such this is an error in your code. see the code snippet below for how it should be
- code snippet
```ruby
super(age, name, parent_permission)
```

Reply:
You are wrong!!!
Those are optional arguments and must named else it will not compile.

Reply:
@chudisoft i was wrong about saying it would throw up errors because after testing your code they don't however on further review, it is not standard to pass in keyword arguments to super, which is why i recommended the code snippet